--- glibc-2.30.orig/Makerules	2019-08-01 13:29:31.000000000 +0900
+++ glibc-2.30/Makerules	2019-11-20 03:00:19.728149811 +0900
@@ -963,6 +963,12 @@ endif
 
 install: check-install-supported
 
+install-lib-all: $(inst_slibdir)/libc.so$(libc.so-version) \
+		$(inst_slibdir)/libc-$(version).so \
+		$(inst_libdir)/libc.so \
+		$(inst_libdir)/libc.a \
+		install-lib
+
 install: $(installed-libcs)
 $(installed-libcs): $(inst_libdir)/lib$(libprefix)%: lib $(+force)
 	$(make-target-directory)
